CONTRIBUTING.md

```diff
@@ -0,0 +1,108 @@
+- [Github Flow](#github-flow)
+- [Commit Lint](#commit-lint)
+
+## Issue Reporting Guidelines
+
+- Always use [ issue templates ](https://github.com/ltelab/disdrodb/issues/new/choose)
+- If you don't find a corresponding issue template please use the template to ask a new template
+
+## Pull Request Guidelines
+
+### We Develop with Github
+We use github to host code, to track issues and feature requests, as well as accept pull requests.
+
+### We Use [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow), So All Code Changes Happen Through Pull Requests
+Pull requests are the best way to propose changes to the codebase (we use [Github Flow](https://docs.github.com/en/get-started/quickstart/github-flow)). We actively welcome your pull requests:
+
+## Python
+- We follow the pep8 and the python-guide writing style
+  - https://docs.python-guide.org/writing/style/
+
+### maybe todo ?
+1. Fork the repo and create your branch from `dev`.
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs or added breaking change, update the documentation and notify the users.
+4. Ensure the test suite passes.
+5. Make sure your code lints.
+6. Issue that pull request!
+
+
+### Recommendation for the pull request
+
+- Add screenshots or GIFs for any UI changes. This will help the person reviewing your code to understand what you’ve changed and how it works.
+  - _Hint: use [Kap](https://getkap.co/) or [Licecap](https://www.cockos.com/licecap/) to record your screen._
+- If your team uses a particular template for pull requests, fill it. Otherwise at least make sure you have:
+  - the user problem you are solving;
+  - acceptance criteria of the ticket;
+  - testing you have done or plan to do before release;
+  - any pull request that are dependent on this one, or any tickets on which this pull request depends.
+
+- The `main` branch is just a snapshot of the latest stable release. All development should be done in dedicated branches. **Do not submit PRs against the `main` branch.**
+
+- Checkout a topic branch from the relevant branch, e.g. `dev`, and merge back against that branch.
+
+- Work in the `frontend` folder and **DO NOT** checkin `dist` in the commits.
+
+- It's OK to have multiple small commits as you work on the PR - GitHub will automatically squash it before merging.
+
+- If adding a new feature:
+  - Add accompanying test case.
+  - Provide a convincing reason to add this feature. Ideally, you should open a suggestion issue first and have it approved before working on it.
+  - Present your issue in the 'discussion' part of this repo
+
+- If fixing bug:
+  - If you are resolving a special issue, add `(fix #xxxx[,#xxxx])` (#xxxx is the issue id) in your PR title for a better release log, e.g. `update entities encoding/decoding (fix #3899)`.
+  - Provide a detailed description of the bug in the PR. Live demo preferred.
+  - Add appropriate test coverage if applicable.
```

Comment:
I would add something to conventions on how to name branches. Here a proposition: 
- Add a reader:  `reader-<institute>-<campaign>`
- Fix a bug: `bugfix-<some_key>-<word>`
- Improve the doc: `doc-<some_key>-<word>`
- Add a new feature: `feature-<some_key>-<word>`
- Refactor some code: `refactor-<some_key>-<word>`
- Optimize some code: `optimize-<some_key>-<word>`